trading_calendars/trading_calendar.py

```diff
@@ -340,7 +398,7 @@ def is_session(self, dt):
         """
         return dt in self.schedule.index
 
-    def is_open_on_minute(self, dt):
+    def is_open_on_minute(self, dt, ignore_breaks=False):
```

Comment:
Shall we add some test coverage for `ignore_breaks=True` and add it to the docstring here?